train_segmentation.py

```diff
@@ -458,7 +460,8 @@ def main(params, config_path):
     overlap = params["sample"]["overlap"]
     min_annot_perc = get_key_def('min_annotated_percent', params['sample']['sampling_method'], 0, expected_type=int)
     num_bands = params['global']['number_of_bands']
-    samples_folder_name = f'samples{samples_size}_overlap{overlap}_min-annot{min_annot_perc}_{num_bands}bands'  # FIXME: won't check if folder has datetime suffix (if multiple folders)
```

Comment:
I propose to use a config parameter to avoid confusion with the samples folder names that share parameters but not image sources.

Reply:
I totally agree. How would you name the samples folder then?

Reply:
currently I implemented it by using the new mlflow_run_name parameter (when given), not entirely sure this is the best way